perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -936,6 +936,13 @@ sub process_report : Private {
             $report->non_public( 1 );
         }
 
+        #Get a list of custom form fields we set and add them into @extra
+        if ($c->cobrand->form_extras){
```

Comment:
As per http://mysociety.github.io/coding-standards.html, please include spaces after the `#` and before the `{`s.